sale_automatic_workflow/README.rst

```diff
@@ -0,0 +1,23 @@
+Sale Automatic Workflow
+=======================
+
+Create workflows with more or less automatization and apply it on sales
+orders.
+
+A workflow can:
+
+- Apply default values:
+  * Packing Policy (partial, complete)
+  * Shipping Policy (prepaid, manual, postpaid, picking)
+  * Invoice On (ordered quantities, shipped quantities)
+  * Set the invoice's date to the sale order's date
+  * Set a sales team
+
+- Apply automatic actions:
+  * Validate the order (only if paid, always, never)
+  * Create an invoice
+  * Validate the invoice
+  * Confirm the picking
+
+This module is used by Magentoerpconnect and Prestashoperpconnect.
+It is well suited for other E-Commerce connectors as well.
```

Comment:
Could you apply the OCA's template?  https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst